recipes/kibana/Dockerfile

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
That's what the official kibana recipe does. It does not provide the steps to build kibana itself, so we did not try to be more catholic than the pope (yet).